portfolio.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <a
+        class="logo"
+        href="./index.html"
+        >WebStudio</a
+      >
+
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="./index.html"
+              >студія</a
+            >
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+              >портфоліо</a
+            >
+          </li>
+          <li class="nav-item">
+            <a
+              class="nav-link"
+              href="#"
+              >контакти</a
+            >
+          </li>
+        </ul>
+      </nav>
+
+      <div class="contacts">
+        <a
+          class="contacts-link"
+          href="mailto:info@devstudio.com"
+          >info@devstudio.com</a
+        >
+        <a
+          class="contacts-link"
+          href="tel:+38 096 111 11 11"
+          >+38 096 111 11 11</a
+        >
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="section buttons">
+        <ul class="buttons-list">
+          <li class="buttons-item button">
+            <button
+              class="buttons-button"
+              type="button"
+            >
+              Усі
+            </button>
+          </li>
+          <li class="buttons-item button">
+            <button
+              class="buttons-button"
+              type="button"
+            >
+              Веб-сайти
+            </button>
+          </li>
+          <li class="buttons-item button">
+            <button
+              class="buttons-button"
+              type="button"
+            >
+              Додатки
+            </button>
+          </li>
+          <li class="buttons-item button">
+            <button
+              class="buttons-button"
+              type="button"
+            >
+              Дизайн
+            </button>
+          </li>
+          <li class="buttons-item button">
+            <button
+              class="buttons-button"
+              type="button"
+            >
+              Маркетинг
+            </button>
+          </li>
+        </ul>
+      </section>
+
+      <section class="section practices">
```

Comment:
use the same  class name, as you used for your images - `projects`, not `practices`